Samples/tutorials/excel-tutorial/My Office Add-in/src/dialogs/popup.js

```diff
@@ -0,0 +1,13 @@
+(function () {
+    "use strict";
+
+    Office.onReady()
```

Comment:
Again, tutorial should use consistency and this should match the same pattern as Office.onReady in taskpane.js.